### PR TITLE
Update release notes draft to version v1.24.0-alpha.1

### PR DESCRIPTION
**Pull request by AuraSinis**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.24.0-alpha.1

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by csantanapr** [comment]
>/assign @parul5sahoo @RinkiyaKeDad @Lp-Francois @csantanapr

@AuraSinis  I wonder why `/assign` didn't work, it didn't add us as assigned in Github

File: releases/release-1.24/release-notes/maps/pr-101719-map.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pr: 101719
+releasenote:
+  text: The deprecated flag `--really-crash-for-testing` is removed.
```
Comment (csantanapr) [inline releases/release-1.24/release-notes/maps/pr-101719-map.yaml:3]:
```suggestion
  text: The deprecated flag `--really-crash-for-testing` was removed.
```

File: releases/release-1.24/release-notes/maps/pr-106179-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 106179
+releasenote:
+  text: Fix bug in error messaging for basic-auth and ssh secret validations.
```
Comment (csantanapr) [inline releases/release-1.24/release-notes/maps/pr-106179-map.yaml:3]:
```suggestion
  text: Fixed bug in error messaging for basic-auth and ssh secret validations.
```

File: releases/release-1.24/release-notes/maps/pr-97252-map.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+pr: 97252
+releasenote:
+  text: Docker runtime support using dockershim in the kubelet is now completely removed
+    in 1.24. The kubelet used to have a module called dockershim which implements
+    CRI support for Docker and it has seen maintenance issues in the Kubernetes community.
```
Comment (csantanapr) [inline releases/release-1.24/release-notes/maps/pr-97252-map.yaml:5]:
```suggestion
    CRI support for Docker, and it has seen maintenance issues in the Kubernetes community.
```

File: releases/release-1.24/release-notes/maps/pr-97252-map.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+pr: 97252
+releasenote:
+  text: Docker runtime support using dockershim in the kubelet is now completely removed
+    in 1.24. The kubelet used to have a module called dockershim which implements
```
Comment (csantanapr) [inline releases/release-1.24/release-notes/maps/pr-97252-map.yaml:4]:
```suggestion
    in 1.24. The kubelet used to have a module called dockershim, which implements
```

**Comment by cici37** [comment]
/assign

Hi Lucas, you may want to add a OWNERS file under //releases/release-1.24/release-notes. It could be a separate PR from this one :)

**Comment by csantanapr** [comment]
testing assign
/assign

**Pull request merged**